packages/react/src/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { extractSprinkles } from "../sprinkles";
+import * as styles from "./Breadcrumb.css";
+import { useBreadcrumbItems } from "./useBreadcrumbItems";
+
+export type BreadcrumbProps = BoxProps<
+  "nav",
+  {
+    children: React.ReactNode;
+    maxItems?: number;
+    separator?: React.ReactNode;
+  }
+>;
+
+export const Breadcrumb = forwardRef<HTMLElement, BreadcrumbProps>(
+  (
+    { children, className, maxItems = Infinity, separator = ">", ...props },
+    ref,
+  ) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
+    const visibleItems = useBreadcrumbItems({
+      children,
+      maxItems,
+      separator,
+    });
+
+    return (
+      <Box asChild {...styles.breadcrumb({}, className)} {...sprinkleProps}>
+        <nav aria-label="Breadcrumb" ref={ref} {...restProps}>
+          <ol {...styles.breadcrumbList()}>{visibleItems}</ol>
```

Comment:
I think we need to use `box` as composition here.
```suggestion
         <Box aschild {...styles.breadcrumbList()}>
             <ol >{visibleItems}</ol>
          </Box>
```